doc/source/tutorial/linalg_batch.md

```diff
@@ -0,0 +1,153 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+orphan: true
+---
+
+```{eval-rst}
+.. jupyterlite:: ../_contents/linalg_batch.ipynb
+   :new_tab: True
+```
+
+(linalg_batch)=
+# Batched Linear Operations
+
+Some of SciPy's linear algebra functions support N-dimensional array input. These operations have not been mathematically generalized to higher-order tensors; rather, the indicated operation is performed on a *batch* (or "stack") of input scalars, vectors, and/or matrices.
+
```

Comment:
Re-reading this, how about adding a small tl;dr explanation around here + stressing that the following narrative text is a detailed explanation.

Consider adding something similar to

In a nutshell, an array of shape `(4, 3, 2)` can be interpreted as a batch of four 3-by-2 matrices (then we say that the the *core shape* is (3, 2) and the *batch shape* is `(4,)`) or as a `(4,3)`batch of 2D vectors (then the *core shape* is `(2,)` and the *batch shape* is `(4, 3)`). The length of the core shape is also referred to as the *core dimension*.
See below for more details.

Obviously, it's up to you --- if feel it's good as is, fine by me.

Reply:
Added this to the "Batch Support" section of the main `linalg` tutorial, which is what links to the full tutorial.

Reply:
Did this address your comments @ev-br?

Reply:
Yes it did, thank you!